packages/translations/package.json

```diff
@@ -33,7 +33,7 @@
     },
     "peerDependencies": {
         "prop-types": "^15.6.2",
-        "react": "^16.5.1"
+        "react": "^16.5.1 || ^17.0.0"
```

Comment:
Can you bump the minimum to 16.8.0?

Reply:
Looks like you already did in #1147

Reply:
Yeah sorry about that, I was upgrading one repo to npm 7 and it was actually throwing installation errors.